resolwe/flow/executors/local.py

```diff
@@ -57,3 +57,7 @@ def terminate(self, data_id):
         time.sleep(self.kill_delay)
         if proc.poll() is None:
             proc.kill()
+
+    def post_register_hook(self):
```

Comment:
Hmm... isn't this redundant since it is the same as in `BaseFlowExecutor`?

Reply:
Good catch!  I'll remove it.